packages/livekit-server-sdk/src/TwirpRPC.ts

```diff
@@ -5,7 +5,15 @@ import type { JsonValue } from '@bufbuild/protobuf';
 
 // twirp RPC adapter for client implementation
 
+type Options = {
+  /** Prefix for the RPC requests */
+  prefix?: string;
+  /** Timeout for fetch requests, in seconds. Must resolve to a positive integer millisecond value. */
```

Comment:
it says milliseconds here, default further down is seemingly in seconds. 
Would be good if the parameter/variable name would carry the expected unit for a more obvious and consistent usage